awx/ui/client/lib/components/layout/_index.less

```diff
@@ -83,8 +83,6 @@
     }
 
     &-side {
-        background: @at-color-side-nav-background;
```

Comment:
as mentioned in slack, unsure if this could mean an only partially grey side nav area.